src/main/java/org/jabref/gui/fieldeditors/MarkdownEditor.java

```diff
@@ -0,0 +1,39 @@
+package org.jabref.gui.fieldeditors;
+
+import javax.swing.undo.UndoManager;
+
+import javafx.scene.control.TextInputControl;
+import javafx.scene.input.Clipboard;
+
+import org.jabref.gui.autocompleter.SuggestionProvider;
+import org.jabref.logic.integrity.FieldCheckers;
+import org.jabref.model.entry.field.Field;
+import org.jabref.preferences.PreferencesService;
+
+import com.vladsch.flexmark.html2md.converter.FlexmarkHtmlConverter;
+
+public class MarkdownEditor extends SimpleEditor {
+
+    private final FlexmarkHtmlConverter flexmarkHtmlConverter = FlexmarkHtmlConverter.builder().build();
+
+    public MarkdownEditor(Field field, SuggestionProvider<?> suggestionProvider, FieldCheckers fieldCheckers, PreferencesService preferences, UndoManager undoManager) {
+        super(field, suggestionProvider, fieldCheckers, preferences, true, undoManager);
+    }
+
+    @Override
+    protected TextInputControl createTextInputControl() {
+        return new EditorTextArea() {
+            @Override
+            public void paste() {
+                Clipboard clipboard = Clipboard.getSystemClipboard();
```

Comment:
I would prefer to use our ClipBoardManager and extend it with a method for gettin html

Reply:
Done at [`c168c9b` (#10896)](https://github.com/JabRef/jabref/pull/10896/commits/c168c9b1977dc52b223f5a2e6f4e0e615ab82eb5).